src/pages/ContentScripts/features/repo-fork-tooltip/ForkChart.tsx

```diff
@@ -137,6 +138,20 @@ const ForkChart = (props: ForkChartProps): JSX.Element => {
     let chartDOM = divEL.current;
     const instance = echarts.getInstanceByDom(chartDOM as any);
     if (instance) {
+      instance.on('dataZoom', (params: any) => {
+        const option = instance.getOption() as {
+          xAxis: { minInterval?: any }[];
+        };
+        const startValue = params.batch[0].start;
+        let minInterval: number;
+        if (startValue != 0) {
```

Comment:
StartValue represents the percentage of the starting position for scaling. When it is not equal to 0, it means that it is not in the initial state and is performing a scaling action, setting minInterval to the month. When it is equal to 0, set minInterval to year.

Reply:
I made further changes. StartValue represents the percentage of the starting position for scaling. EndValue represents the percentage of the end position of scaling. When startValue equals 0 and endValue equals 100, it indicates that it is in the initial state and minInterval is set to the year. Otherwise, it means that a scaling or dragging action has been taken and minInterval has been set to the month。

Reply:
I see what you mean. But this cannot fix the issue essentially. An example could be this new repo [MetaGPT](https://github.com/geekan/MetaGPT). It is a newly created project and it doesn't have years' of records, the axis labels are missing because we set the `minInterval` to year. 

The idea to change the `minInterval` should be fine, but we need to find some other attributes in options or do calculation on the startValue/endValue to know when to change the value.

Reply:
Sorry, I didn't consider everything thoroughly. Thank you for your reminder. I will make corrections.